.changes/add-attribute-for-drag-region.md

```diff
@@ -0,0 +1,45 @@
+---
+"tauri": "minor"
+---
+
+Added
+attribute `data-tauri-drag-region-container`, `data-tauri-drag-region-title`, `data-tauri-drag-region-interactive`, to
+enable drag on children, control maximize on double click, control is or not interactable on button/input/etc.
+
+Description:
+
+`data-tauri-drag-region`: mark this element as "drag region"
+
+`data-tauri-drag-region-container`: mark the "drag region" effects its children
+
+`data-tauri-drag-region-title`: mark the "drag region" maximize on double click (default add when
+only `data-tauri-drag-region` is set)
+
+`data-tauri-drag-region-interactive`: mark the "drag region" wouldn't prevent event to its children that interact-able,
+like button, input, etc. (check by "is property 'value' exists")
```

Comment:
In my design, exclude single element is done by set `data-tauri-drag-region` to false,
and if `data-tauri-drag-region-container` is also used, then also exclude it's children

the `data-tauri-drag-region-interactive` is for mark a container that shouldn't prevent events to default interact-able element.

Reply:
yeah I understand but this feels quite unpredictable for me, I'd rather use an explicit attribute to mark an element as not draggable instead of adding to the parent.

Reply:
yeah I understand but this feels quite unpredictable for me, I'd rather use an explicit attribute to mark an element as not draggable instead of adding to the parent.